tools/build-neatvnc.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Update and install dependencies
+sudo apt update
+sudo apt install -y meson ninja-build libpixman-1-dev zlib1g-dev libdrm-dev pkgconf
+
+CURRENT_DIR=$(pwd)
+# Clone and build aml
+git clone https://github.com/any1/aml && cd aml
+meson -Dprefix=$CURRENT_DIR/_static build
+meson configure build --default-library=static
```

Comment:
Simplify the command. Use `meson -Dprefix=/tmp/_static -Ddefault_library=static build` instead.